### ISSUE TITLE
hystrix pkg don't support fallback func

### ISSUE THREAD

**Issue by wwangbo** [go-kit/kit#1227]
### What would you like?

I want to use hystrix as cb and want to set fallback func once execute failed

**Comment by peterbourgon** [comment]
You can do this without explicit support in Go kit, can't you?

**Comment by wwangbo** [comment]
Yes, I can, but since our Go-Kit supports this function, we'd better support this function. Let me mention MR, is it ok for you to support this function?

**Comment by peterbourgon** [comment]
Feel free to make a PR but consider it a method of explanation, as I don't yet understand why this is important.
